### PR TITLE
Add Multiple Hunks To Test #33

### PR DESCRIPTION
**Pull request by iAlexeze**
### Summary

This pull request introduces multiple hunks in the `example.diff` file to support comprehensive testing of issue #33 . 
This change becomes important as the initial contents of the `example.diff` do not show a file with several hunks (#38 ), and we need to ensure that our project can accurately demonstrate and test scenarios involving more than one hunk.

### Changes

- Updated the `example.diff` file to include multiple hunks.
- Ensured that the project can now accurately demonstrate and test scenarios involving more than one hunk.

_This update is part of our ongoing efforts to improve the testing capabilities of our project._ 

### Output: [⬇️](https://emojipedia.org/down-arrow)
2hunks
3hunks

Please verify that the changes in this pull request allow for the successful testing of issue #33 .
Thank you.

@Julow

### CODE REVIEW

File: example.diff [hunk 1]
```diff
@@ -159,4 +159,29 @@ index e69de29..eee417f 100644
...
++Take a tour at https://ocaml.org/docs/tour-of-ocaml
++Write your first OCaml program here - https://ocaml.org/docs/your-first-program
+diff --git a/dir3/outreachy.txt b/dir3/outreachy.txt
+index 0xchj65..e03c6s4
+--- a/dir3/outreachy.txt 
++++ b/dir3/outreachy.txt
+@@ -0,0 +1,2 @@
++Outreachy provides internships in open source and open science. 
++Outreachy provides internships to people subject to systemic bias and impacted by underrepresentation in the technical industry where they are living.
+@@ -5,6 +5,7 @@
```
Comment (Julow) [inline example.diff:181]: The hunks do not show any context (common lines). How did you generate it?

**Review by Julow — COMMENTED** [review]
It does successfully test https://github.com/panglesd/diffcessible/pull/33

File: example.diff (hunk 1 above)
Comment (iAlexeze) [reply example.diff:181]: Okay, it's hand written. Perhaps I should have it this way:


        -Outreachy provides internships in open source and open science. 
        +Outreachy provides internships to people subject to systemic bias and impacted by underrepresentation in the technical industry where they are living.

Any better ideas please?
@Julow

File: example.diff [hunk 2]
```diff
@@ -159,4 +159,29 @@ index e69de29..eee417f 100644
...
+@@ -0,0 +1,2 @@
++OCaml is an industrial-strength functional programming language with an emphasis on expressiveness and safety.
++Learning OCaml is interesting.
+@@ -3,4 +3,4 @@
+-Want to begin?
++Take a tour at https://ocaml.org/docs/tour-of-ocaml
++Write your first OCaml program here - https://ocaml.org/docs/your-first-program
+diff --git a/dir3/outreachy.txt b/dir3/outreachy.txt
+index 0xchj65..e03c6s4
+--- a/dir3/outreachy.txt
```
[the remark this answers is not in the document]
Comment (iAlexeze) [reply example.diff:176]: Will apply this in the next commit.
Thank you.

File: example.diff (hunk 1 above)
Comment (Julow) [reply example.diff:181]: What do you think of writing two files that would generate the wanted diffs with `git diff --no-index file1 file2` ?

Comment (iAlexeze) [reply example.diff]: Thanks a lot for this, I have generated the diffs as recommended.

Ready for another review.
@Julow

**Review by Julow — APPROVED** [review]
Looks good! Thanks :) I'll merge.

**Pull request merged**